### PR TITLE
SVA 427 Add log in/out buttons to settings

### PR DESCRIPTION
**Pull request by mirkiy**
- [Description](#description)
  - [Type of change](#type-of-change)
- [Testing locally](#testing-locally)
- [Checklist](#checklist)

# Description

- Please include a summary of the change and which issue is fixed.
- List any dependencies that are required for this change.

Fixes # (issue number)

## Type of change

Please delete options that are not relevant

- [] Bug fix (non-breaking change which fixes an issue)
- [] New Feature (non-breaking change which adds functionality)
- [] Breaking change(fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

# Testing locally

- Add here anything someone would need to do to test your code locally, or delete this section if nothing needed
- E.g. run `npm install` in the app, refresh the database, look at a particular project that has a video, add a `.env` file value (NB do not share any sensitive details here as PRs are public)

# Checklist

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have removed any unnecessary comments or console logging
- [] I have made corresponding changes to the documentation (if required)
- [] I have addressed accessibility, if needed
- [] I have followed best practices, e.g. NativeBase approaches and theming
- [] I have checked the app in dark mode, if making front-end design changes
- [] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [] I have updated the version numbers in `package.json` files in the [app](DEPLOYMENT.md#app-deployment) and/or [api](DEPLOYMENT.md#api-deployment-on-aws) directories as needed

### CODE REVIEW

**Review by PammyLo — COMMENTED** [review]
Great job @mirkiy! Looks and works fine. Just a few little observations. 

Side note - there appear to be contradictory designs regarding the colour of the button when a user is logged in, the design for the screen shows the error.100 red as you've used here but the design for the STA button component which the ticket also links to shows the primary pink and a pressed state. Perhaps a wee liaise with design team over this?

File: src/NativeBase/Containers/SettingsContainer.tsx [hunk 1]
```diff
@@ -87,19 +101,31 @@ const SettingsContainer = () => {
...
-        space={4}
-        padding={4}
-      >
-        <VStack space="4">
+      <VStack flex={1} padding={4} space={4}>
+        <Heading textAlign="center" size="lg" marginTop={8} fontFamily="title">
+          SETTINGS
+        </Heading>
+
+        <View style={{ marginTop: 48 }}>
```
Comment (PammyLo) [inline src/NativeBase/Containers/SettingsContainer.tsx:123]: Style is not need here, just marginTop={'48px'} will suffice :)

File: src/NativeBase/Containers/SettingsContainer.tsx [hunk 2]
```diff
@@ -130,9 +156,25 @@ const SettingsContainer = () => {
           </Checkbox> */}
         </VStack>
 
-        <VStack alignItems="center" space="2">
+        <VStack alignItems="center" space={2} width="100%" paddingBottom={4}>
           <Text fontSize="xs">Version {version}</Text>
           <PrivacyAndTermsLinks />
+          {featureFlags.login ? (
```
[the remark this answers is not in the document]
Comment (mirkiy) [reply src/NativeBase/Containers/SettingsContainer.tsx]: This has been fixed now, thank you :)

File: .expo/packager-info.json
[the remark this answers is not in the document]
Comment (mirkiy) [reply .expo/packager-info.json:1]: Thanks for this, fixed now too :)

File: src/NativeBase/Containers/SettingsContainer.tsx (hunk 1 above)
Comment (mirkiy) [reply src/NativeBase/Containers/SettingsContainer.tsx]: Thank fixed now too :)

**Review by PammyLo — APPROVED** [review]

**Review by DougieWougie — APPROVED** [review]

**Pull request merged**